pkg/limayaml/defaults.go

```diff
@@ -171,6 +172,15 @@ func defaultGuestInstallPrefix() string {
 //   - CACertificates Files and Certs are uniquely appended in d, y, o order
 func FillDefault(y, d, o *LimaYAML, filePath string) {
 	instDir := filepath.Dir(filePath)
+
+	var existingLimaVersion string // empty if the instance was created with Lima prior to v0.20
```

Comment:
Maybe add a comment that it can also be empty when editing a template file without an instance, i.e. `limactl edit foo.yaml`.

That's why `existingLimaVersion` should always be used with `isExistingInstanceDir` below.

I just noticed that we could also check `filepath.Base(filePath) == "lima.yaml"`, but `isExistingInstanceDir` only receives the directory part, so it won't work. I think the current check is probably good enough.

Reply:
done